adcm_client/packer/data/config_data.py

```diff
@@ -9,6 +9,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+# pylint: disable=consider-using-with
```

Comment:
This warning should be fixed and not disabled.

Reply:
Fixed in last few commits